.gitignore

```diff
@@ -73,5 +73,6 @@ terraform.rc
 
 # generated mock files
 /graveler/committed/**/mock/
+/pyramid/mock/*
```

Comment:
need to remove the file pyramid/mock/pyramid.go first

Reply:
Done.